src/destination/selector.js

```diff
@@ -85,13 +89,17 @@ export default function destinationSelectorFactory($container, config) {
         defaultConfig
     )
         .setTemplate(selectorTpl)
-        .on('init', function() {
+        .on('init', function () {
             this.render($container);
```

Comment:
**suggestion:** the `uniqId` needs to be set in the config
```suggestion
            this.config.uniqId = uuid(); // need to add this too: `import uuid from 'lib/uuid';`
            this.render($container);
```

Reply:
> import uuid from 'lib/uuid';

fixed and added the import line to imports area :) thanks!